optimum/intel/ipex/modeling_base.py

```diff
@@ -60,7 +64,11 @@
 logger = logging.getLogger(__name__)
 
 
-_IPEX_SUPPORT_MODEL_TYPES = ("llama",)
+_IPEX_SUPPORT_MODEL = {
+    "llama": ("text-generation",),
+    "bert": ("question-answering",),
+    "vit": ("image-classification",),
+}
```

Comment:
same remark, a model type based conversion seems more adequate as the task variants of models only differ on a "model head" level, with the same core architecture.

Reply:
fixed